scripts/download-odf-nightly.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+# Download a nightly build of odf cli executable for the current platform and
+# architecture.
+#
+# For available builds see:
+# https://quay.io/repository/rhceph-dev/mcg-cli?tab=tags
+
+set -e
+
+build=${1:?Usage $0 BUILD}
+
+version=4.19.0
+platform=$(uname)
+
+case $platform in
+Darwin)
+    platform=macosx
+    executable=odf
+    ;;
+Linux)
+    platform=linux
+    machine=$(uname -m)
+    case $machine in
+    x86_64)
+        executable=odf-amd64
+        ;;
+    aarch64)
+        executable=odf-arm64
+        ;;
+    ppc64le,s390x)
+        executable=odf-$machine
+        ;;
+    *)
+        echo "unsupported machine $machine"
+        exit 1
+        ;;
+    esac
+    ;;
+*)
+    echo "unsupported platform $platform"
+    exit 1
+    ;;
+esac
+
+oc image extract quay.io/rhceph-dev/mcg-cli:$version-$build --file /usr/share/odf/$platform/$executable
```

Comment:
I learned that we have this image for the latest release:
https://quay.io/repository/rhceph-dev/odf4-odf-cli-rhel9?tab=tags&tag=v4.19

Testing it now.